Assignment_05.rb

```diff
@@ -85,6 +85,49 @@ def validate_time_logs
     end
     true
   end
+  
+  #This method validate the input format
+  def validate_input_format
+    if id==nil || id=="deafult"
+      puts "Input format validation failed - A Task must contain an id"
+      return false
+    elsif name==nil || name=="deafult"
```

Comment:
@AritraChatterjee-qi can you please use inbuilt method `nil?` instead of using `== nil`. also we need to give proper spacing around any operator.

Reply:
also is there any reason to check with id  `deafult` ? I think we can remove it.

Reply:
okay making those changes

Reply:
I have check id and name with "default" 

> also is there any reason to check with id `deafult` ? I think we can remove it.

Firstly sorry for the typo, It should be default.
Secondly, In the Task constructor I have set the default value of id and name as 'default' and time_logs as []. That's why that check is need. For example if in case the Task constructor is called with one or two parameter.